### PR TITLE
bump moonbase to runtime 2801

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR bumps the moonbase alpha runtime to 2801
Goes with https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/429

### Checklist

- [x] I have added a label to this PR 🏷️

### CODE REVIEW

File: variables.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 networks:
   development:
     build_tag: v0.36.0
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2800-latest
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-latest
```
Comment (albertov19) [inline variables.yml:4]:
```suggestion
    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-7ae4
```

File: variables.yml [hunk 2]
```diff
@@ -15,10 +15,10 @@ networks:
     hex_chain_id: '0x507'
     chain_spec: alphanet
     block_explorer: https://moonbase.moonscan.io/
-    spec_version: runtime-2700
+    spec_version: runtime-2801
     parachain_release_tag: v0.36.0 # must be in this exact format for links to work
     parachain_sha256sum: a4d143b6fd3c4fc43341df5a0e9a608180fee1b2e5c2ade12c4bcb4da49fa93a
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2800-latest
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-latest
```
Comment (albertov19) [inline variables.yml:21]:
```suggestion
    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-7ae4
```

File: variables.yml [hunk 3]
```diff
@@ -397,7 +397,7 @@ networks:
     node_directory: /var/lib/moonriver-data
     parachain_release_tag: v0.36.0
     parachain_sha256sum: a4d143b6fd3c4fc43341df5a0e9a608180fee1b2e5c2ade12c4bcb4da49fa93a
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2800-latest
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-latest
```
Comment (albertov19) [inline variables.yml:400]:
```suggestion
    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-7ae4
```

File: variables.yml [hunk 4]
```diff
@@ -743,7 +743,7 @@ networks:
     node_directory: /var/lib/moonbeam-data
     parachain_release_tag: v0.36.0
     parachain_sha256sum: a4d143b6fd3c4fc43341df5a0e9a608180fee1b2e5c2ade12c4bcb4da49fa93a
-    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2800-latest
+    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-latest
```
Comment (albertov19) [inline variables.yml:746]:
```suggestion
    tracing_tag: moonbeamfoundation/moonbeam-tracing:v0.36.0-2801-7ae4
```

File: builders/interoperability/xcm/core-concepts/weights-fees.md [hunk 5]
```diff
@@ -236,7 +236,7 @@ For each XCM instruction, the weight units are converted to balance units as par
...
+| [{{ networks.moonbeam.xcm.instructions.wei_per_weight.display }}](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbeam.spec_version}}/runtime/moonbeam/src/lib.rs#L141){target=\_blank} | [{{ networks.moonriver.xcm.instructions.wei_per_weight.display }}](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonriver.spec_version}}/runtime/moonriver/src/lib.rs#L143){target=\_blank} | [{{ networks.moonbase.xcm.instructions.wei_per_weight.display }}](https://github.com/moonbeam-foundation/moonbeam/blob/{{networks.moonbase.spec_version}}/runtime/moonbase/src/lib.rs#L148){target=\_blank} |
```
Comment (albertov19) [inline builders/interoperability/xcm/core-concepts/weights-fees.md:239]: Wow how do you check this?

**Review by albertov19 — CHANGES_REQUESTED** [review]
LGTM - But I suggest changing the tracing image to the one that we share in the comms. Maybe @wuzhong-papermoon can submit this next time as he will take care of the comms from now on

File: builders/interoperability/xcm/core-concepts/weights-fees.md (hunk 5 above)
Comment (eshaben) [reply builders/interoperability/xcm/core-concepts/weights-fees.md:239]: I just search for `moonbase.spec_version` and double-check that the links are still pointing to the right line. At this point, I know what I'm looking for so it doesn't take me too long

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**